projects/kit/src/lib/masks/date-time/preprocessors/valid-date-time-preprocessor.ts

```diff
@@ -66,7 +66,7 @@ export function createValidDateTimePreprocessor({
 
         validatedValue += validatedDateString;
 
-        const paddedMaxValues = padTimeSegments(DEFAULT_TIME_SEGMENT_MAX_VALUES);
+        const paddedMaxValues = padEndTimeSegments(DEFAULT_TIME_SEGMENT_MAX_VALUES);
```

Comment:
```suggestion
        const paddedMaxValues = padStartTimeSegments(DEFAULT_TIME_SEGMENT_MAX_VALUES);
```

Reply:
This change does not affect anything, but it will be useful for code clarity.